server/fleet/app.go

```diff
@@ -488,9 +488,6 @@ func (c *AppConfig) Obfuscate() {
 	for _, zdIntegration := range c.Integrations.Zendesk {
 		zdIntegration.APIToken = MaskedPassword
 	}
-	for _, calIntegration := range c.Integrations.GoogleCalendar {
-		calIntegration.PrivateKey = MaskedPassword
-	}
```

Comment:
We should somehow mask the `api_json_key`.

Reply:
Noah instructed not to mask it. We can add masking later. Any specific concern?

Reply:
Consistency?
Why are we masking SMTP/JIRA/Zendesk passwords but not masking the Google calendar private key?

(I don't recall what user roles can access the app config and why we decided to mask SMTP/JIRA/Zeendesk.)

Reply:
Yes, I brought it up. Seems like it was a mistake to mask those. Also, those passwords cannot be edited via GUI, but this setting can be.